examples/tutorials/3d-perspective-image.py

```diff
@@ -4,6 +4,15 @@
 
 Create 3-D perspective image or surface mesh from a grid
 using :meth:`pygmt.Figure.grdview`.
+
+.. note::
+
+    This tutorial assumes the use of a Python notebook, such as IPython or Jupyter Notebook.
+    To see the figures while using a Python script instead, use
+    ``fig.show(method="external)`` to display the figure in the default PDF viewer.
```

Comment:
```suggestion
    ``fig.show(method="external)`` to display the figure in the default image viewer.
```
Is the default PDF or PNG?

Reply:
> ```suggestion
>     ``fig.show(method="external)`` to display the figure in the default image viewer.
> ```
> Is the default PDF or PNG?

The Figure.show() documentation makes it seem like it is the PDF viewer.

Reply:
Ah ok, all good then.